spec/book_spec.rb

```diff
@@ -0,0 +1,10 @@
+require './book'
+describe Book do
+  let(:book) { Book.new('Title', 'Author') }
+  let(:person) { double('Person') }
+  let(:date) { '2023-11-23' }
+  it 'should initialize correctly' do
+    expect(book.title).to eq('Title')
+    expect(book.author).to eq('Author')
+  end
+end
```

Comment:
- [ ] Well done, you did a good job by implementing the few changes in the `test` files. But it seems like you overlooked the `previous` code reviewers comment. As per the instructions, you should implement or `add` the test for all methods of the class. Even the last `code reviewer` emphasized on the adding the test for `add_rental` method but you still haven't fulfilled that requirement. Kindly `add` the test case for the method, that you created inside the `book.rb` file.